### PR TITLE
add support for friends routes; minor client and readme clean-up

### PR DESCRIPTION
**Pull request by crazy4groovy**
Gets watched friends from an account

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -23,16 +23,7 @@ class HttpClient {
     return body
   }
 
-  _request (options) {
-    options.headers = Object.assign(
```
Comment (crazy4groovy) [inline lib/client.js]: just moved these into `request` function

File: lib/client.js [hunk 2]
```diff
@@ -43,24 +34,19 @@ class HttpClient {
...
-      headers
-    )
+    Object.assign(headers, { Accept: 'application/json', 'User-Agent': USER_AGENT })
 
     if (this.accessToken) {
-      qs = Object.assign(
-        { access_token: this.accessToken },
-        qs
-      )
+      Object.assign(qs, { access_token: this.accessToken })
```
Comment (crazy4groovy) [inline lib/client.js]: `assign` automatically mutates the object for us.

**Comment by crazy4groovy** [comment]
Would you like to take a look at this PR?

File: README.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ damn.getNotifications
 Returns the current user's watch feed:
 
 ```js
-damn.getWatchFeed
+damn.getWatchFeed()
```
Comment (n6g7) [inline README.md]: Great, thanks for catching these!

**Comment by crazy4groovy** [comment]
Ah right, that was a mistake I caught after the PR. Code should be fixed now. Apparently a coveralls CI error? Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}

**Comment by n6g7** [comment]
Awesome, thanks again @crazy4groovy! :D

**Pull request merged**